src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -26,7 +26,11 @@ public class OtlpExporterOptions : IOtlpExporterOptions
 {
     internal const string DefaultGrpcEndpoint = "http://localhost:4317";
     internal const string DefaultHttpEndpoint = "http://localhost:4318";
+#if NET462_OR_GREATER || NETSTANDARD2_0
+    internal const OtlpExportProtocol DefaultOtlpExportProtocol = OtlpExportProtocol.HttpProtobuf;
+#else
     internal const OtlpExportProtocol DefaultOtlpExportProtocol = OtlpExportProtocol.Grpc;
```

Comment:
Maybe we should consider marking `OtlpExporterProtocol.Grpc` obsolete for .NET Framework so that users may get a compiler warning. This would provide another means to communicate to users that they should really shift to using OTLP/HTTP.

```csharp
[ObsoleteAttribute("CAUTION: OTLP/gRPC is no longer supported for .NET Framework or .NET Standard targets without supplying a properly configured HttpClientFactory. It is strongly encouraged that you migrate to using OTLP/HTTP.")]
```